ios/Package.swift

```diff
@@ -86,6 +91,13 @@ var package = Package(
             name: "Repository",
             dependencies: []
         ),
+        .target(
+            name: "SettingFeature",
+            dependencies: [
+                .target(name: "Styleguide"),
+                .target(name: "Component")
```

Comment:
Please sort this 🙏🏼 
```suggestion
                .target(name: "Component"),
                .target(name: "Styleguide"),

```

Reply:
I wonder how to ordering? I mean import order is descending like the following.👀

Reply:
Yes, I mean ascending order 🙏🏼 

Following is target dependency sorting rule.
```swift 
dependencies: [
  // targets (ascending order) .target()
  // SPM targets (ascending order) .product()
]
```

Reply:
Thank you for showing rule, I see!